test/recipe-test.js

```diff
@@ -326,11 +326,18 @@ describe('search recipes', () => {
     assert.deepEqual(filteredRecipes, []);
   });
 
-  it('should return all recipes if only an empty string is entered', () => {
+  it('should do nothing if only an empty string is entered', () => {
```

Comment:
subtle

Reply:
This is the reason I need the double conditional instead of the `?` in your earlier comment